config/config.go

```diff
@@ -264,3 +246,7 @@ func init() {
 	// struct tag translation etc.
 	validate = validator.New()
 }
+
+func init() {
+	initialize()
+}
```

Comment:
nit: can you put this at the top of the file? It's easier to find that way.

Reply:
https://github.com/onflow/flow-go/pull/4523/commits/49e9aa5d71c4a17abe34e7d354dfc8138e821dea